server/storage/mvcc/watcher.go

```diff
@@ -22,9 +22,14 @@ import (
 	"go.etcd.io/etcd/api/v3/mvccpb"
 )
 
-// AutoWatchID is the watcher ID passed in WatchStream.Watch when no
-// user-provided ID is available. If pass, an ID will automatically be assigned.
-const AutoWatchID WatchID = 0
+const (
+	// InvalidWatchID represents an invalid watch ID and prevents duplication with an existing watch.
+	InvalidWatchID WatchID = -1
+
+	// AutoWatchID is the watcher ID passed in WatchStream.Watch when no
+	// user-provided ID is available. If pass, an ID will automatically be assigned.
+	AutoWatchID WatchID = 0
```

Comment:
Please consider to define the two const in [clientv3](https://github.com/etcd-io/etcd/blob/main/client/v3/watch.go), so as to remove the `clientv3` dependency on `server`

Reply:
Are these two constants defined both on the client and server?